application/testing/CMakeLists.txt

```diff
@@ -44,7 +49,7 @@ function(f3d_test)
   add_test(NAME "f3d::${F3D_TEST_NAME}" COMMAND $<TARGET_FILE:f3d> ${F3D_TEST_ARGS} COMMAND_EXPAND_LISTS)
 
   set(_timeout "30")
-  if(F3D_TEST_LONG_TIMEOUT OR F3D_TEST_INTERACTION OR F3D_TEST_HDRI)
+  if(F3D_TEST_LONG_TIMEOUT OR F3D_TEST_INTERACTION)
```

Comment:
Can you remember to me why interaction tests are running slower?

Reply:
They just do. They are just running slow. Like one interaction every second.

Reply:
weird